settings.php

```diff
@@ -863,6 +871,7 @@ function get_FTL_data($arg)
 							<label for="zip_file">File input</label>
 							<input type="file" name="zip_file" id="zip_file">
 							<p class="help-block">Upload only Pi-hole backup files.</p>
+							<input type="hidden" name="token" value="<?php echo $token ?>">
```

Comment:
This sends the token only for `Import` but not `Export` (23 lines further up). Also note that this form's action is `scripts/pi-hole/php/teleporter.php` which does not check the CSRF token at all